packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -35,7 +34,7 @@ contract OptimisticGovernor is Module, Lockable {
         bytes32 indexed identifier,
         uint256 indexed timestamp,
         bytes ancillaryData,
-        SkinnyOptimisticOracleInterface.Request request
+        OptimisticOracleInterface.Request request
```

Comment:
Should this parameter just be dropped entirely?

Reply:
yep